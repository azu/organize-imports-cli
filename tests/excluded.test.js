import { test, expect } from '@jest/globals';
import runCli from "./run-cli.js";

test("exclude in tsconfig - excludes files correctly", async () => {
  const result = await runCli("cli/exclude", ["--list"]);
  expect(result.status).toMatchSnapshot();
  expect(result.stdout).toMatchSnapshot();
  expect(result.stderr).toMatchSnapshot();
  expect(result.write).toMatchSnapshot();
});
