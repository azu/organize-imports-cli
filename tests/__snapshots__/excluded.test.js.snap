// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exclude in tsconfig - excludes files correctly 1`] = `1`;

exports[`exclude in tsconfig - excludes files correctly 2`] = `""`;

exports[`exclude in tsconfig - excludes files correctly 3`] = `"appendStdout is not defined"`;

exports[`exclude in tsconfig - excludes files correctly 4`] = `[]`;
